UnityProject/Assets/Scripts/Managers/ObjectManager.cs

```diff
@@ -1,8 +1,4 @@
 ﻿using UnityEngine;
-using System.Collections;
-using System.Collections.Generic;
-using PlayGroup;
-using UI;
```

Comment:
I dislike the way you keep cutting out these without your current ask requiring you too. Just don't, you could be introducing error's and no one would know it was this onrelated PR.

Reply:
They are unused in the script so they are automatically removed. I'd have to rework my VS to be shit and not remove UNUSED inheritance